src/components/theme/Loader/index.jsx

```
@@ -1,6 +1,6 @@
 import React from "react";
 
-export const Spinner = ({ size = "md" }) => {
+export const Spinner = ({ size = "md", className }) => {
```

ESLint

[error] 'size' is missing in props validation

(react/prop-types)